tbtadm/controller.cpp

```diff
@@ -434,7 +434,9 @@ void tbtadm::Controller::topology()
     {
         auto last = host.first == controllers.rbegin()->first;
         m_out << host.second.m_desc[0];
-        indentation += last ? indentLast : indent;
+
+        indentation = last ? indentLast : indent;
```

Comment:
= instead of += breaks the printing of a chain of devices

Reply:
This is AFAIK first level of indentation, so after you have added "|    " it would be hanging there forever. For the last controller we do not need it. I will add multi device controller test.

Reply:
Oh, sorry, you are correct, of course. This is about the controller itself.
Adding such a test will be nice anyway :)
And this needs rebasing before merging it (and probably squashing the last commit into the one that changed `read` to `readAndTrim`).